src/main/scala/ai/privado/utility/SQLParser.scala

```diff
@@ -210,7 +212,7 @@ object SQLNodeBuilder {
       val columnNode = NewSqlColumnNode()
         .name(queryColumn.name)
         .code(queryColumn.name)
-        .lineNumber(queryLineNumber + queryColumn.lineNumber - 1)
+        .lineNumber(queryColumn.lineNumber)
```

Comment:
@khemrajrathore why we were adding `queryLineNumber` to column's linenumber?

Reply:
To get the effective line number of the columnNode, in the file

Reply:
queryColumn.linenumber will give the linenumber of column node w.r.t to the passed query, But we need the line number w.r.t file

Reply:
@hiteshbedre

Reply:
Commit: 8c24bbdfc59ec27a7bd48753821c17f269946ef9 exclusively refactored lineNumber.
Logic:
1. While creating column node also passed Table node's object.
2. While initiating lineNumber if column added table's lineNumber to it.